src/components/Verify.js

```diff
@@ -322,7 +322,6 @@ const Verify = (props) => {
                 </Paper>
                 <OptimizedImage
                   src={capture.imageUrl}
-                  width={400}
```

Comment:
I messaged Statically.io about the problem.  If they can't fix that, then we may need to look for a different tool.  In the meantime I've added the width back to the Verify thumbnails and the CaptureDetailDialog image so that the rotation of the images match but left it off for the Capture Match tool

We can hold off on this PR if this doesn't seem like a workable temporary solution.

Reply:
Thanks. Let's merge the changes you've made – I think it's a definite improvement.